### PR TITLE
document how to ignore files using command

### PR DESCRIPTION
**Pull request by scytacki**
This also upgrades the version of the wait-for-it script since recent versions of alpine changed the timeout command

### CODE REVIEW

**Comment by scytacki** [comment]
@knowuh can you take a look at this?

File: README.md [hunk 1]
```diff
@@ -75,14 +75,38 @@ FROM alpine:edge
...
+    command: ["-ignore", "Name *.log"]
+    volumes:
+      - sync-volume:/data
+      - .:/host_data
+...
+```
+More info about the syntax for ignoring files is here:
+https://www.cis.upenn.edu/~bcpierce/unison/download/releases/stable/unison-manual.html#ignore
+
+If you are wanting to ignoring a folder on your host system, you might be able to optimize performance further by adding an additional
```
Comment (knowuh) [inline README.md]: small suggestion: "wanting to ignoring" → "want to ignore"

Is it  possible to ignore multiple paths, files, or patterns using this method, if so, how would that be done?  Are the images default ignores ignored?

**Review by knowuh — APPROVED** [review]
Thanks for this documentation.

It's better than what we have. I had some questions and a small comment I left inline, but nothing that should block this PR.

Thanks!

**Review by knowuh — APPROVED** [review]
Thank you for these additions, they are really clear.

**Pull request merged**